cmd/node.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/module"
+)
+
+type Node interface {
+	module.ReadyDoneAware
```

Comment:
Node is implemented as `Startable` in #1559. I'm not opposed to adding a `NodeRunner` to handle the node lifecycle, but I think that should be in a different PR since there's already a lot of change here.

Reply:
Yeah, I think a different PR makes sense too. Can we just make sure we create an issue to track it? Thanks